src/data/faq.json

```diff
@@ -95,15 +95,15 @@
                                 "anchor": "ramp_down_mailbox",
                                 "active": false,
                                 "textblock": [
-                                    "Until May 31, 2023 there will be community support, in which incoming messages will be read and answered if applicable."
+                                    "The community support, which includes reading and answered if applicable to messages received at <a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>, will continue until May 31, 2023. After that, the mailbox <a href='CoronaWarnApp...RKI...de' class='email'>CoronaWarnApp...RKI...de</a> will still be available, which is planned to be maintained until the end of 2023."

```

Comment:
"The community support, which includes reading and answered if applicable to messages received at"
=> 
"The community support, which includes reading and answering messages (if applicable) received by"

Reply:
```suggestion
                                    "The community support, which includes reading and answering messages (if applicable) received by <a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>, will continue until May 31, 2023. After that, the mailbox <a href='CoronaWarnApp...RKI...de' class='email'>CoronaWarnApp...RKI...de</a> will still be available, which is planned to be maintained until the end of 2023."
```

Reply:
Fixed with [`86a47bc` (#3511)](https://github.com/corona-warn-app/cwa-website/pull/3511/commits/86a47bc365f962c5cff3dd5fb157bdb69ed1d02d).